branding/src/main/scala/com/normation/plugins/branding/BrandingPluginDef.scala

```diff
@@ -39,22 +39,28 @@ package com.normation.plugins.branding
 
 import bootstrap.liftweb.Boot
 import com.normation.rudder.AuthorizationType.Administration
-import net.liftweb.http.ClasspathTemplates
+import com.normation.rudder.domain.logger.PluginLogger
+import net.liftweb.http.{ClasspathTemplates, ResourceServer}
 import net.liftweb.sitemap.Loc.LocGroup
 import net.liftweb.sitemap.Loc.Template
 import net.liftweb.sitemap.Loc.TestAccess
 import net.liftweb.sitemap.LocPath.stringToLocPath
 import net.liftweb.sitemap.Menu
-
 import com.normation.plugins.PluginStatus
 import com.normation.plugins.DefaultPluginDef
 
 class BrandingPluginDef(override val status: PluginStatus) extends DefaultPluginDef {
 
   override val basePackage = "com.normation.plugins.branding"
 
+  def init = {
+    PluginLogger.info(s"loading '${buildConf.getString("plugin-name")}:${version.toString}' plugin")
+    // resources in src/main/resources/toserve must be allowed:
+    ResourceServer.allow{
+      case "branding" :: _ => true
```

Comment:
All the context of init should be worthless now, both actions are done in Rudder for all plugins.